src/github/deployment.ts

```diff
@@ -75,6 +75,18 @@ export const processDeployment = async (
 
 	logger.info("deployment message transformed");
 
+	const jiraClient = await getJiraClient(
```

Comment:
Same comment as in branch.ts -> makes more sense to keep the jiraPayload + conditional together.